docker/Dockerfile

```
@@ -0,0 +1,74 @@
+WORKDIR /home/airflow
+RUN git clone https://mirror.ghproxy.com/github.com/deepmodeling/dpti.git
+
+RUN pip config set global.index-url https://pypi.tuna.tsinghua.edu.cn/simple
+RUN cd dpti && pip install .
+
+USER root
+RUN apt-get -y install procps
+
+ENV AIRFLOW_HOME=/home/airflow/airflow/
+
+RUN service postgresql start && service postgresql status \
```

Hadolint (2.12.0)

[error] Do not use sudo as it leads to unpredictable behavior. Use a tool like gosu to enforce root

(DL3004)